Terminal.py

```diff
@@ -63,8 +63,8 @@ def linux_terminal():
     if wm:
         # elementary OS: `/usr/lib/gnome-session/gnome-session-binary --session=pantheon`
         # Gnome: `gnome-session` or `gnome-session-binary`
-        # Linux Mint Cinnamon: `cinnamon-session --session cinnamon`
-        if wm[0].startswith('gnome-session') or wm[0].startswith('cinnamon-session'):
+        # Linux Mint Cinnamon: `cinnamon-sessio cinnamon-session --session cinnamon
```

Comment:
Might want to restore the trailing backtick.

Reply:
Ah, that's what I get for copy/pasting around lines when setting up the PR =/

```suggestion
        # Linux Mint Cinnamon: `cinnamon-sessio cinnamon-session --session cinnamon`
```